packages/react-components/react-overflow/src/components/OverflowItem/OverflowItem.test.tsx

```diff
@@ -0,0 +1,69 @@
+import { MenuTrigger, MenuProvider, type MenuContextValue } from '@fluentui/react-menu';
+import { render } from '@testing-library/react';
+import * as React from 'react';
+
+import { OverflowItem } from './OverflowItem';
+
+describe('OverflowItem', () => {
+  describe('ref', () => {
+    it('handles ref', () => {
+      const ref = jest.fn();
+
+      render(
+        <OverflowItem id="test" ref={ref}>
+          <div id="child" />
+        </OverflowItem>,
+      );
+      expect(ref).toHaveBeenCalledTimes(1);
+      expect(ref).toHaveBeenCalledWith(expect.objectContaining({ id: 'child', tagName: 'DIV' }));
+    });
+
+    it('handles ref propagation', () => {
+      const itemRef = jest.fn();
+      const childRef = jest.fn();
+
+      render(
+        <OverflowItem id="test" ref={itemRef}>
+          <MenuTrigger>
+            <div id="child" ref={childRef} />
+          </MenuTrigger>
+        </OverflowItem>,
+      );
+
+      expect(itemRef).toHaveBeenCalledTimes(1);
+      expect(itemRef).toHaveBeenCalledWith(expect.objectContaining({ id: 'child', tagName: 'DIV' }));
+
+      expect(childRef).toHaveBeenCalledTimes(1);
+      expect(childRef).toHaveBeenCalledWith(expect.objectContaining({ id: 'child', tagName: 'DIV' }));
+    });
+
+    it('handles ref propagation in reverse order', () => {
+      const itemRef = jest.fn();
+      const childRef = jest.fn();
+      const triggerRef = jest.fn();
+
+      const menuContextValue: Partial<MenuContextValue> = {
+        triggerRef: triggerRef as unknown as React.MutableRefObject<HTMLElement>,
+      };
+
+      render(
+        <MenuProvider value={menuContextValue as MenuContextValue}>
+          <MenuTrigger>
+            <OverflowItem id="test" ref={itemRef}>
+              <div id="child" ref={childRef} />
+            </OverflowItem>
+          </MenuTrigger>
+        </MenuProvider>,
+      );
```

Comment:
We have similar problem with `Overflow` component:
```jsx
<Overflow ref={overflowRef}>
  <div id="container" ref={containerRef} />
</Overflow>
```

The `containerRef` is never set. Would it make sense to fix that in this PR as well?

Reply:
@miroslavstastny sure, fixed it